docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md

```diff
@@ -193,13 +193,15 @@ You can use similar commands to view the `cpu_request`, `mem_limit` and
 The following information is available to Containers through environment
```

Comment:
If there are fieldRefs for the container information as well,  I would prefer the format of this section be something like:

The following `fieldRef`s are available to Containers through environment
variables and DownwardAPIVolumeFiles:

* `spec.nodeName` - Name of the Node.
* `status.hostIP` - IP of the Node.
* ...

Reply:
those are resourceRefs, with a different format I'm not as familiar with